api/pablo/internal/pablo_manager.py

```diff
@@ -24,6 +24,13 @@
 from pablo.store.schema import ImageVariantsTable
 
 
+IPFS_PROVIDER_PREFIXES = [
+    'https://ipfs.io/ipfs/',
+    'https://ipfs.infura.io/ipfs/',
```

Comment:
this isnt going to be enough, for some of them (e.g. infura and later even pablo) we will need to provide credentials

Reply:
i imagine it will need to be a config object we create. or maybe we should create a `IpfsRequester` and pass in a list to this class. the requester can have the default headers and a default url and just expose ipfs related methods